packages/captp/jsconfig.json

```diff
@@ -3,17 +3,17 @@
   "compilerOptions": {
     "target": "esnext",
     "module": "esnext",
-
+    "checkJs": true,
     "noEmit": true,
-/*
-    // The following flags are for creating .d.ts files:
-    "noEmit": false,
-    "declaration": true,
-    "emitDeclarationOnly": true,
-*/
     "downlevelIteration": true,
     "strictNullChecks": true,
     "moduleResolution": "node",
+    "types": []
```

Comment:
Curious what motivated this. It seems about right, but I would not have thought of changing this.

Reply:
Without it there are [errors from `node_modules/events`](https://github.com/endojs/endo/actions/runs/3162294420/jobs/5148757446),
```
../../node_modules/events/events.js:53:3 - error TS2323: Cannot redeclare exported variable 'defaultMaxListeners'.

 53   Object.defineProperty(EventEmitter, 'defaultMaxListeners', {
      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 54     enumerable: true,
    ~~~~~~~~~~~~~~~~~~~~~
...
 64     }
    ~~~~~
 65   });
    ~~~~

../../node_modules/events/events.js:67:3 - error TS2323: Cannot redeclare exported variable 'defaultMaxListeners'.

67   EventEmitter.defaultMaxListeners = defaultMaxListeners;
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

../../node_modules/events/events.js:165:11 - error TS2339: Property 'context' does not exist on type 'Error'.

165       err.context = er;
              ~~~~~~~
```